### PR TITLE
Update link in README.md

### LINKED ISSUE #31: [OTHER] Update Awesome GitHub profiles link
### What would you like to share?

The message/link to the Awesome GitHub profiles repository should be updated to contain a proper message. https://github.com/EddieHubCommunity/gh-actions-html-table-generator/pull/26#discussion_r886845191

https://github.com/EddieHubCommunity/gh-actions-html-table-generator/blob/d422d55a1e16f2e5be4b0ef6324af1b6f1b66a71/README.md#L98

### Steps to contribute

_If you've already forked the repository, skip to step 3_

1. Fork the repository [**here**](https://github.com/EddieHubCommunity/gh-actions-html-table-generator/fork).
2. Make a new branch named `update_message`.
3. Open the `README.md` file.
4. Replace line 98 content with the following content:

```markdown
From this repository usage: [Awesome GitHub profiles](https://github.com/EddieJaoudeCommunity/awesome-github-profiles).
```

5. Commit and push your changes.
6. Create a [**pull request**](https://github.com/EddieHubCommunity/gh-actions-html-table-generator/compare).
7. You're done now! 🎉 You just have to wait until the maintainers check your PR.

### Additional information

This is a good first issue. Please leave it open to first-time contributors and beginners.
If you need any help with this issue, let us know! 🙂

### PR DESCRIPTION
**Pull request by joypoddar**
## Fixes Issue

Closes #31 

## Changes proposed

* Update link in README.md

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Kindly look at the commit message and let me know if there is any changes to be made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,29 +70,30 @@ jobs:
...
-      with:
-        json-file-path: 'data.json'
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        columns: 3
-        object-field-names: '[ "githubUsername", "name", "imageUrl", "issueNumber" ]'
-        file-to-use: 'README.md'
-        html-cell: '<td align="center"><p><a href="https://github.com/{{ githubUsername }}">{{ name }}</a></p><img src="{{ imageUrl }}" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/{{ issueNumber }}">(:100: give your vote)</a></p></td>'
+      - uses: actions/checkout@v2
+      - name: Read/Write data into README
+        uses: eddiejaoude/github-actions-reader-writer@v0.1
```
Comment (Panquesito7) [inline README.md]: I think we should probably update the links here.
```suggestion
        uses: EddieHubCommunity/gh-actions-html-table-generator@main
```

File: README.md [hunk 2]
```diff
@@ -70,29 +70,30 @@ jobs:
...
+        with:
+          json-file-path: "data.json"
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          columns: 3
+          object-field-names: '[ "githubUsername", "name", "imageUrl", "issueNumber" ]'
+          file-to-use: "README.md"
+          html-cell: '<td align="center"><p><a href="https://github.com/{{ githubUsername }}">{{ name }}</a></p><img src="{{ imageUrl }}" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/{{ issueNumber }}">(:100: give your vote)</a></p></td>'
```
Comment (Panquesito7) [inline README.md]:
```suggestion
          html-cell: '<td align="center"><p><a href="https://github.com/{{ githubUsername }}">{{ name }}</a></p><img src="{{ imageUrl }}" /><p><a href="https://github.com/EddieHubCommunity/awesome-github-profiles/issues/{{ issueNumber }}">(💯 give your vote)</a></p></td>'
```

File: README.md [hunk 3]
```diff
@@ -70,29 +70,30 @@ jobs:
...
 ```typescript
 [
-    {
-        "name": "Akas Rai",
-        "githubUsername": "akasrai",
-        "imageUrl": "https://user-images.githubusercontent.com/624760/88123456-d40df580-cbc2-11ea-9add-a7fc8675b243.png",
-        "issueNumber": 12
-    }
-]
+  {
```
Comment (Panquesito7) [inline README.md:89]: The indentation here seems to be wrong. The previous one was fine, IMO.
I believe other instances should be updated to have the correct indentation as well.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joypoddar) [reply README.md]: https://github.com/EddieHubCommunity/gh-actions-html-table-generator/issues/31
There is no mention of the suggested change in the issue above.

File: README.md (hunk 3 above)
Comment (joypoddar) [reply README.md:89]: I will have to check the indentation. Is there a guide?

Comment (Panquesito7) [reply README.md:89]: > I will have to check the indentation. Is there a guide?

Not necessarily. For JSON, it's usually a 2-line indentation.

File: README.md (hunk 1 above)
Comment (Panquesito7) [reply README.md]: Right. Should probably be fixed in another PR.

File: README.md (hunk 3 above)
Comment (joypoddar) [reply README.md:89]: Thank you

File: README.md (hunk 2 above)
Comment (joypoddar) [reply README.md]: @Panquesito7 I added the suggested changes. The checks have passed. Can you take a look and see if all the criteria have been met?

File: README.md [hunk 4]
```diff
@@ -70,29 +70,30 @@ jobs:
...
+    name: "Akas Rai",
+    githubUsername: "akasrai",
+    imageUrl:
+      "https://user-images.githubusercontent.com/624760/88123456-d40df580-cbc2-11ea-9add-a7fc8675b243.png",
+    issueNumber: 12,
+  },
+];
```
Comment (Panquesito7) [inline README.md:96]: AFAIK, this comma is going to make the JSON formatting complain about it.
```suggestion
  }
]
```

File: README.md (hunk 2 above)
Comment (Panquesito7) [inline README.md]: This link is still not updated yet.

File: README.md [hunk 5]
```diff
@@ -70,29 +70,30 @@ jobs:
...
-    - name: Read/Write data into README
-      uses: eddiejaoude/github-actions-reader-writer@v0.1
-      with:
-        json-file-path: 'data.json'
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        columns: 3
-        object-field-names: '[ "githubUsername", "name", "imageUrl", "issueNumber" ]'
-        file-to-use: 'README.md'
-        html-cell: '<td align="center"><p><a href="https://github.com/{{ githubUsername }}">{{ name }}</a></p><img src="{{ imageUrl }}" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/{{ issueNumber }}">(:100: give your vote)</a></p></td>'
+      - uses: actions/checkout@v2
```
Comment (Panquesito7) [inline README.md]: We might want to stick to the newer version, which is faster and better than the previous one.
```suggestion
      - uses: actions/checkout@v3
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by joypoddar** [comment]
I made all the suggested corrections... What is the next step forward?

File: README.md [hunk 6]
```diff
@@ -70,29 +70,30 @@ jobs:
...
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          columns: 3
+          object-field-names: '[ "githubUsername", "name", "imageUrl", "issueNumber" ]'
+          file-to-use: "README.md"
+          html-cell: '<td align="center"><p><a href="https://github.com/{{ githubUsername }}">{{ name }}</a></p><img src="{{ imageUrl }}" /><p><a href="https://github.com/EddieHubCommunity/awesome-github-profiles/issues/{{ issueNumber }}">(💯 give your vote)</a></p></td>'
 ```
 
 ### Json file
 
 ```typescript
```
Comment (Panquesito7) [inline README.md]: Should probably be `javascript` here instead.
```suggestion
```javascript
```

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Comment by joypoddar** [comment]
> Looks good. Thanks! tada

@Panquesito7 Thank you for being so patient. I learnt a lot and more importantly, did away with my fear of getting into open-source. I am contributing actively to an open-source called ReactPlay.

**Comment by Panquesito7** [comment]
> > Looks good. Thanks! tada
> 
> @Panquesito7 Thank you for being so patient. I learnt a lot and more importantly, did away with my fear of getting into open-source. I am contributing actively to an open-source called ReactPlay.

That is _so_ awesome to hear! Keep going! 🚀
Do let us know if you need any help. 🙂

**Comment by joypoddar** [comment]
> > > Looks good. Thanks! tada
> > 
> > 
> > @Panquesito7 Thank you for being so patient. I learnt a lot and more importantly, did away with my fear of getting into open-source. I am contributing actively to an open-source called ReactPlay.
> 
> That is _so_ awesome to hear! Keep going! rocket Do let us know if you need any help. slightly_smiling_face

Thank you!! Just wanted to know when the PR will be merged.

File: README.md [hunk 7]
```diff
@@ -48,7 +48,7 @@ with:
 
 ```yaml
 with:
-    json-file-path: 'your-filename.json'
+  json-file-path: "your-filename.json"
```
Comment (eddiejaoude) [inline README.md:51]: line 33 uses single quotes, so why does this one (and others) change to double quotes? Consistency is important

**Comment by eddiejaoude** [comment]
Great collaboration 👍

**Comment by joypoddar** [comment]
@Panquesito7 I am getting these errors
image
What should be done?

**Comment by Panquesito7** [comment]
> @Panquesito7 I am getting these errors image What should be done?

I think you need to `git pull` first, make your changes, then push.
Or you can resolve the conflicts manually, then push your changes (see [GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line)).

**Comment by joypoddar** [comment]
@Panquesito7 I have three options for the soln.
1. git config pull.rebase false  # merge
2. git config pull.rebase true   # rebase
3. git config pull.ff only       # fast-forward only

Is there any organisational/repository preference or do I go ahead with the 1st option.

**Comment by Panquesito7** [comment]
> @Panquesito7 I have three options for the soln.
> 
> 1. git config pull.rebase false  # merge
> 2. git config pull.rebase true   # rebase
> 3. git config pull.ff only       # fast-forward only
> 
> Is there any organisational/repository preference or do I go ahead with the 1st option.

I'd use the 1st option, it's the best one, IMO.

**Comment by joypoddar** [comment]
@Panquesito7 Do we have any pending tasks before merging?

**Comment by Panquesito7** [comment]
> @Panquesito7 Do we have any pending tasks before merging?

AFAIK, nope. I do not have write access, only someone with that will be able to merge the PR. 🙂

File: README.md [hunk 8]
```diff
@@ -30,7 +30,7 @@ This is json, and contains a list of the names of the fields in your json file d
 
 ```yaml
 with:
-    object-field-names: '[ "firstname", "lastname" ]'
+  object-field-names: "[ "firstname", "lastname" ]"
```
Comment (eddiejaoude) [inline README.md:33]: I think this will break?

Comment (joypoddar) [reply README.md:33]: do we need single quote or double?

**Pull request closed without merging**